__tests__/app.test.js

```diff
@@ -97,6 +97,60 @@ describe("/api", () => {
                     })
                 })
             })
+            describe("?sort_by=query", () => {
+                test('Returns sorted list', () => {
+                    return request(app)
+                    .get("/api/articles?sort_by=title")
+                    .expect(200)
+                    .then(({body}) => {
+                        const { articles } = body
+                        expect(articles).toBeSortedBy("title", {descending: true})
+                    })
```

Comment:
You want to have a test that responds with an article filtered by topic and check that all the objects only have the queried topic. Make sure they are checking the length to ensure there aren't any false positives too 👀